### PR TITLE
fixed the issues with contribution.md

### PR DESCRIPTION
**Pull request by bhargavshirin**

### CODE REVIEW

**Comment by bhargavshirin** [comment]
there were some typos. fixed it

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ the community that you are working on it.
 3. For convention help, we provide a [pre-commit](https://pre-commit.com/hooks.html) file.
    This tool will run before each commit and will automatically reformat code or raise warnings and errors based on the
    code format or Python typing.
-   You can install and setup it up by doing:
+   You can install and set it up by doing:
```
Comment (jrobinAV) [inline CONTRIBUTING.md:70]: Please revert this, as it does not seem to be proper English. No?

**Comment by bhargavshirin** [comment]
Done

**Comment by bhargavshirin** [comment]
@jrobinAV

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ the community that you are working on it.
 3. For convention help, we provide a [pre-commit](https://pre-commit.com/hooks.html) file.
    This tool will run before each commit and will automatically reformat code or raise warnings and errors based on the
    code format or Python typing.
-   You can install and setup it up by doing:
+   You can install and setup it by doing:
```
Comment (jrobinAV) [inline CONTRIBUTING.md]: I am not an English native, but my understanding is that "setup" is a noun, while the verbal form is "set up".
 
```suggestion
   You can install and set it up by doing:
```

**Review by jrobinAV — APPROVED** [review]

**Comment by jrobinAV** [comment]
Thank you for your contribution.

**Pull request merged**